test/message-based/PriceAware.test.ts

```diff
@@ -144,6 +144,47 @@ describe("Price Aware - editable assembly version", function () {
         expect(price).to.equal(BigNumber.from("500000000"));
     });
 
+    it("should return correct prices in the specified order", async function () {
+        let ORACLE_PRICES = (forTime: number) => ({
+            prices: [
+                {symbol: "ETH", value: 10},
+                {symbol: "AVAX", value: 5},
+                {symbol: "LINK", value: 2}
+            ],
+            timestamp: forTime - 3000
+        });
+
+        sample = WrapperBuilder
+            .mockLite(sample)
+            .using(ORACLE_PRICES);
+
+        await sample.authorizeProvider();
+
+        await syncTime(); // recommended for hardhat test
+        let prices = await sample.executeWithPrices([toBytes32("AVAX"), toBytes32("LINK"), toBytes32("ETH")]);
+        let expectedResult = [BigNumber.from("500000000"), BigNumber.from("200000000"), BigNumber.from("1000000000")];
+
+        expect(prices[0].toNumber()).to.eq(expectedResult[0].toNumber());
```

Comment:
Can we refactor the same lines (e.g. `expect(prices[i].toNumber()).to.eq(expectedResult[0].toNumber());`) using `for` loop